Homework_5.py

```diff
@@ -0,0 +1,108 @@
+# #Homework_5
+import datetime as dt
+import time
+
+
+class PostStory:  # create a class for writing posts in file
+    def __init__(self, post_name, post_text, post_param, calculated_parem='', string_for_post=''):
+        self.post_name = post_name  # type of post: News, PrivatAd, Weather Forecast
+        self.post_text = post_text  # the text of post
+        self.post_param = post_param  # input parameter, which is different for each post
+        self.calculated_parem = calculated_parem  # calculated parameter, which is different for each post
+        self.string_for_post = string_for_post  # string, which is published for each post
```

Comment:
Why just not use make_string function?:) Maybe just check if all the data for make_string is presented in the object